block/stream/output/state_changes.proto

```diff
@@ -288,7 +288,7 @@ message SingletonUpdateChange {
         /**
          * A change to the exchange rates singleton.
          * <p>
-         * This change SHALL be present if the HBAR`<=>`USD exchange
+         * This change SHALL be present if the HBAR:USD exchange
```

Comment:
We could keep the original with entities and still work with javadoc (another PBJ limitation that it doesn't handle these; markdown javadoc _cannot_ come soon enough, in my opinion).
```suggestion
         * This change SHALL be present if the <tt>HBAR&lt;=&gt;USD</tt> exchange
```